src/lib/translation.ts

```diff
@@ -29,14 +29,15 @@ export async function getTranslation(string: string, options: TranslationOptions
     return response.text.join(' ')
   }
 
-  if (options.translationService === TranslationService.deepl) {
+  if (options.translationService === TranslationService.deepl || options.translationService === TranslationService.deeplFree) {
```

Comment:
I would use some shorter and easier to understand constants for these, quite long, lines. My suggestion would be:
```
const params = new URLSearchParams()

const isDeepL = options.translationService === TranslationService.deepl
const isDeepLFree = options.translationService === TranslationService.deeplFree
const isYandex = options.translationService === TranslationService.yandex

if (isYandex) { ... }

if (isDeepL || isDeepLFree) { ... }
```